test/Microsoft.AspNet.Mvc.Core.Test/ModelBindingHelperTest.cs

```diff
@@ -51,7 +51,7 @@ public class ModelBindingHelperTest
         public async Task TryUpdateModel_ReturnsFalse_IfModelValidationFails()
         {
             // Arrange
-            var expectedMessage = TestPlatformHelper.IsMono ? "The field MyProperty is invalid." :
+            var expectedMessage = TestPlatformHelper.IsMono ? "RequiredAttribute_ValidationError" :
```

Comment:
Yes. There is a mono bug. https://github.com/aspnet/External/issues/19. I haven't referred it in all places.